src/pages/dashboard/dashboard.js

```diff
@@ -170,17 +172,17 @@ const Dashboard = ({
                         />
                       </Widget>
                     </Grid>
-                    <Grid item lg={4} md={6} sm={12} xs={12}>
+                    <Grid item lg={4} md={6} sm={12} xs={12} className={classes.donut5Style}>
```

Comment:
Can you write the conditional logic with a ternary operator here instead of in JSS. IF it's in the jss the component will be rendered every time and then hides it.